python/semantic_kernel/kernel.py

```diff
@@ -133,6 +134,97 @@ def register_semantic_function(
 
         return function
 
+    async def run_stream_async(
+        self,
+        func: SKFunction,
```

Comment:
Is this intentionally written to only run a single SK function at a time? It might be a bit unclear to users that you cannot run a collection of functions with this method as opposed to it's `run_async` non-streaming equivalent. 

I think this should have the same function signature as `run_async` below. 

There are of course some decisions to make here. If this method is changed to consume _n_ functions, should only the last one be streamed, or should all?

@shawncal

Reply:
i understand and agree with this should have same function signature as `run_async`
I  changed function signature. 

and originally i wanted to add `SKFunction.invoke_stream_async`. 
If you think this good idea, i can try it too

Reply:
I like the idea of invoke_stream_async on SKFunction. I would just add it to sk_function.py for now, not sk_function_base.